.laminas-ci/install-apcu-extension-via-pecl.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+PHP_VERSION="$1"
+
+if ! [[ "${PHP_VERSION}" =~ 8\.2 ]]; then
+  echo "APCu is only installed from pecl for PHP 8.2, ${PHP_VERSION} detected."
+  exit 0;
+fi
+
+set +e
+apt install make
```

Comment:
Could be removed if we re-add `make` to the container.

Ref: https://github.com/laminas/laminas-continuous-integration-action/issues/135

Reply:
Note that you may want a `-y` on these commands anyway :D

Reply:
I'll release a new CI action, and get rid of this line :+1:

Reply:
024c7da3a0b1f6e057d43bcfaeae730ff4e4d882